### PR TITLE
adding limit flag to verifier

### PR DESCRIPTION
**Pull request by svv232**
adding back the limit verifier flag as an auxiliary to the bit-catchup work

### CODE REVIEW

**Comment by svv232** [comment]
!ci-build-me

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml [hunk 1]
```diff
@@ -1857,11 +1860,14 @@ let internal_commands logger =
               ~conf_dir:(Some conf_dir) ()
           in
           let%bind result =
+            let cap lst =
+              Option.value_map ~default:Fn.id ~f:(Fn.flip List.take) limit lst
```
Comment (georgeee) [inline src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1864]:
```suggestion
              Option.value_map ~default:Fn.id ~f:(Fn.flip List.take) limit
```

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml [hunk 2]
```diff
@@ -1857,11 +1860,14 @@ let internal_commands logger =
               ~conf_dir:(Some conf_dir) ()
           in
           let%bind result =
+            let cap lst =
```
Comment (georgeee) [inline src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1863]:
```suggestion
            let cap =
```

**Review by georgeee — APPROVED** [review]
One NIT left

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml (hunk 2 above)
Comment (svv232) [reply src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1863]: this was the easiest way for the type to resolve as an 'a list

**Pull request merged**